app/_src/guides/builtin-gateway.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Add a builtin gateway 
+---
+
+To get traffic from outside your mesh inside it (North/South) with {{site.mesh_product_name}} you can use 
+a builtin gateway.
+
+In the [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/), traffic was only able to get in the mesh by port-forwarding to an instance of an app
+inside the mesh.
+This is not something that can be used in production and you need to set up a gateway to receive traffic external to the mesh.
+In this guide you will add a gateway in front of the demo-app service to expose it publicly.
+
+{% mermaid %}
+---
+title: service graph of the demo app with a builtin gateway on front
+---
+flowchart LR
+  subgraph edge-gateway
+    gw0(/ :8080)
+  end
+  demo-app(demo-app :5000)
+  redis(redis :6379)
+  gw0 --> demo-app 
+  demo-app --> redis
+{% endmermaid %}
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Start a gateway 
+
+### Create a `MeshGatewayInstance` 
+
+A [`MeshGatewayInstance`](/docs/{{ page.version }}/using-mesh/managing-ingress-traffic/builtin-k8s/) configures the pods
+that will run the gateway.
+
+Create it by running:
+```shell
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: MeshGatewayInstance
+metadata:
+  name: edge-gateway
+  namespace: kuma-demo
+spec:
+  replicas: 2
```

Comment:
Did you mean to have 2 replicas? If so, maybe just make a short note that we are doing that so that if someone is looking in the GUI at the result of these commands and they see 2 of these they don't think "wait... did I apply that command twice by accident?" (or just amend the config to make the single MeshGatewayInstance)

Reply:
Ah you do mention the 2 instances again further down, so maybe thats enough 👍